File: 1.js
function main(){
    var arr = [34,56,2,1,90,67,45,23,11,2,34];
    let size = arr.length;
    for(let i = 0; i < arr.length - 1; i++){
        for(let j = 0; j < arr.length - 1; j++){
            if(arr[j] > arr[j + 1]){
                var temp  = arr[j];
                arr[j] = arr[j+1];
                arr[j+1] = temp;
            }
        }
    }

    for(i = 0; i < arr.length; i++){
        console.log(arr[i]);
    }
}
main();

File: 1.cc
#include<iostream>
using namespace std;

void swap(int *a, int *b){
    int x = *a;
    *a = *b;
    *b = x;
}

int main(){
    
    int arr[] = {34,56,1,23,3,23,90,4,56,3,33,78};
    int size = sizeof(arr)/sizeof(int);
    for(int i = 0; i < size - 1; i++){
        for(int j = 0; j < size - 1; j++){
            if(arr[j] > arr[j+1]){
                swap(&arr[j], &arr[j + 1]);
            }
        }
    }
    for(int i = 0; i < size; i++){
        printf("%d\t", arr[i]);
    }

    return 0;
}